app/models/bill.rb

```diff
@@ -1,2 +1,6 @@
 class Bill < ApplicationRecord
+  has_one  :procedure, foreign_key: :procedure_id
```

Comment:
You don't need to specify the `foreign_key` here. It's good to specify the foreign key constraint in your migrations, but in the model, adding `foreign_key` does something slightly different: it specifies naming conventions that differ from Rails' defaults. In this case, the foreign key is `procedure_id`, which is exactly what Rails expects to look for in the `procedure` table, so you don't need to specify it here.

There's an example of when to use `foreign_key` in a model here: https://guides.rubyonrails.org/association_basics.html#bi-directional-associations